documentation/asciidoc/computers/raspberry-pi/bootflow-2711.adoc

```diff
@@ -73,3 +73,16 @@ Please see the xref:raspberry-pi.adoc#raspberry-pi-4-bootloader-configuration[bo
 === Bootloader Updates
 
 The bootloader may also be updated before the firmware is started if a `pieeprom.upd` file is found. Please see the xref:raspberry-pi.adoc#raspberry-pi-4-boot-eeprom[bootloader EEPROM] page for more information about bootloader updates.
+
+=== Fail-safe OS updates (TRYBOOT)
+
+The bootloader/firmware provide a one-shot flag which, if set, is cleared but causes `tryboot.txt` to be loaded instead of `config.txt`. This alternate firmware would specify the pending OS update firmware, cmdline, kernel and os_prefix parameters. Since the flag is cleared before starting the firwmare a crash or reset will cause the original `config.txt` file to be loaded on the next reboot.
```

Comment:
I still think the "This alternate firmware" should say "This alternate config" ?